package-builders/Dockerfile.ubuntu1604

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:16.04
+
+ARG EMAIL=bot@netdata.cloud
+ARG FULLNAME="Netdata Builder"
+ARG VERSION=0.1
+
+ENV EMAIL=$EMAIL
+ENV FULLNAME=$FULLNAME
+ENV VERSION=$VERSION
+
+# This is needed to keep package installs from prompting about configuration.
+ARG DEBIAN_FRONTEND=noninteractive
+
+ADD package-builders/debian-build.sh /build.sh
+RUN chmod +x /build.sh
```

Comment:
Unnecessary if you `chmod +x` the file and commit it that way into Git :) I prefer doing it this way myself.